plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -324,6 +331,7 @@ func ClusterRoleBindings() []rbac.ClusterRoleBinding {
 		rbac.NewClusterBinding("cluster-admin").Groups(user.SystemPrivilegedGroup).BindingOrDie(),
 		rbac.NewClusterBinding("system:discovery").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
 		rbac.NewClusterBinding("system:basic-user").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
+		rbac.NewClusterBinding("system:cluster-facilities").Groups(user.AllAuthenticated, user.AllUnauthenticated).BindingOrDie(),
```

Comment:
I pushed a fresh (squashed) patch (551ec8ed6ab51753aee72fefbe50bb38693d590c) that still bears the "system:cluster-facilities" role name. That's easy enough to change if you think of a name you'd prefer instead.